src/prod.Dockerfile

```diff
@@ -31,14 +31,19 @@ RUN apk add --no-cache imagemagick && \
 RUN pip install pip --upgrade
 ADD requirements $APP_HOME/requirements
 RUN pip install -r ./requirements/prod.txt
+
+# Copy project
+COPY . $APP_HOME
+
+FROM base AS clean-scenarios
+ENTRYPOINT ["python", "manage.py", "loop_clean_scenarios"]
+
+FROM base AS webserver
```

Comment:
This is the last target in the Dockerfile, which makes it the default target. These commands do the same:

`docker build . --file prod.Dockerfile` 
`docker build . --file prod.Dockerfile --target webserver` 

It is indeed used to build the container tagged with `wagtail`

I've renamed the target to `wagtail` and explicitly added `--target wagtail` to `build-push-images-azure.yml`. Because it's usually better to be explicit.

Reply:
Nice, perfect solution; makes it clear and transparent IMO.